sdk/servicebus/service-bus/test/internal/unit/serviceBusMessage.spec.ts

```diff
@@ -15,168 +15,173 @@ const assert = chai.assert;
-  const expectedLockToken = uuid_to_string(fakeDeliveryTag);
-
-  it("Lock token in peekLock mode", () => {
-    const sbMessage = new ServiceBusMessageImpl(
-      amqpMessage,
-      { tag: fakeDeliveryTag } as Delivery,
-      false,
-      "peekLock"
-    );
-
-    assert.equal(sbMessage.lockToken, expectedLockToken, "Unexpected lock token found");
-  });
-
-  it("Lock token in receiveAndDelete mode", () => {
-    const sbMessage = new ServiceBusMessageImpl(
-      amqpMessage,
-      { tag: fakeDeliveryTag } as Delivery,
-      false,
-      "receiveAndDelete"
-    );
-
-    assert.equal(!!sbMessage.lockToken, false, "Unexpected lock token found");
-  });
-});
-
-describe("ServiceBusMessageImpl AmqpAnnotations unit tests", () => {
-  const message_annotations: MessageAnnotations = {};
-  message_annotations[Constants.enqueuedTime] = Date.now();
-  message_annotations[Constants.partitionKey] = "dummy-partition-key";
-  // message_annotations[Constants.viaPartitionKey] = "dummy-via-partition-key";
-  message_annotations["random-msg-annotation-key"] = "random-msg-annotation-value";
-
-  const delivery_annotations: DeliveryAnnotations = {
-    delivery_annotations_one: "delivery_annotations_one_value",
-    delivery_annotations_two: "delivery_annotations_two_value",
-    delivery_annotations_three: "delivery_annotations_three_value"
-  };
-
-  const amqpMessage: RheaMessage = {
-    body: "hello",
-    message_annotations,
-    delivery_annotations,
-    delivery_count: 2,
-    first_acquirer: true,
-    ttl: 123456,
-    durable: true,
-    priority: 9876,
-    message_id: "random_messageId",
-    reply_to: "random_replyTo",
-    to: "random_to",
-    correlation_id: "random_correlationId",
-    content_type: "random_contentType",
-    content_encoding: "random_contentEncoding",
-    absolute_expiry_time: 123908745,
-    creation_time: 45612349,
-    group_id: "random_groupId",
-    group_sequence: 98723560,
-    reply_to_group_id: "random_replyToGroupId",
-    subject: "random_subject",
-    user_id: "random_user_id"
-  };
-
-  const sbMessage = new ServiceBusMessageImpl(amqpMessage, fakeDelivery, false, "peekLock");
-
-  it("headers match", () => {
-    assert.equal(sbMessage._rawAmqpMessage.header?.firstAcquirer, amqpMessage.first_acquirer);
-    assert.equal(sbMessage._rawAmqpMessage.header?.timeToLive, amqpMessage.ttl);
-    assert.equal(sbMessage._rawAmqpMessage.header?.durable, amqpMessage.durable);
-    assert.equal(sbMessage._rawAmqpMessage.header?.priority, amqpMessage.priority);
-    assert.equal(sbMessage._rawAmqpMessage.header?.deliveryCount, amqpMessage.delivery_count);
-
-    assert.equal(sbMessage.deliveryCount, amqpMessage.delivery_count);
-    assert.equal(sbMessage.timeToLive, amqpMessage.ttl);
-  });
-
-  it("message annotations match", () => {
-    if (!sbMessage._rawAmqpMessage.messageAnnotations) {
-      throw new Error("Message Annotations should not be empty");
+describe("ServiceBusMessageImpl unit tests", () => {
```

Comment:
Why?

Reply:
Wraps the entire set of message tests in a single describe (ie, run this entire file).

Reply:
(I just realized, I didn't actually add a test here)

I originally added a unit test in addition to the live test but then realized I didn't need it. I do still like doing this when I can since it makes it easier to run everything in the file.